### ISSUE TITLE
Change backup method:Flow is not correct.

### ISSUE THREAD

**Issue by cakesoft-swati** [bithyve/hexa#5845]
- If I have cloud backup in app and I want to change the backup method then it is giving me the seed option i.e "Backup with seed".But when I click on "Share Recovery key"button of backup with seed then it is showing me the Confirm seed word backup screen.

![Screenshot_20220609-152323](https://user-images.githubusercontent.com/62699947/172820477-9deb29af-98ad-479f-b452-e176a56bdf23.jpg)

- It should show me the seed words screen first.

**Comment by cakesoft-swati** [comment]
- Still the change flow is not correct.

https://user-images.githubusercontent.com/62699947/173036841-7db7a541-5e8f-4c48-ba83-e1c4f71c5294.mp4
- I changed cloud backup method to seed backup and complete the process.But when I clicked "View Health" button app crashed.
- Also I changed seed backup to cloud backup.But message on home screen didn't change. Also on security center page it was showing seed icon and text was from cloud backup.
![temp (230)](https://user-images.githubusercontent.com/62699947/173038346-f4fb3bac-34a7-4ee1-8bcc-e5095b8f2dca.jpg)
![temp (231)](https://user-images.githubusercontent.com/62699947/173038355-7335e560-8809-4f97-ae90-0b3323082386.jpg)

- After setting the password, it stays on the same page.

https://user-images.githubusercontent.com/62699947/173038667-f52091bf-35ea-4271-a68f-a9a480d06df7.mp4
